stetho/src/main/java/com/facebook/stetho/inspector/elements/android/window/WindowFetcher.java

```diff
@@ -0,0 +1,103 @@
+package com.facebook.stetho.inspector.elements.android.window;
+
+import android.content.Context;
+import android.os.Build;
+import android.view.View;
+import android.view.WindowManager;
+import android.view.WindowManagerGlobal;
+
+import com.facebook.stetho.common.Util;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+
+/*
+ * 4.0.3_r1    WindowManagerImpl   private View[] mViews;
```

Comment:
this documentation is useful, but please also try to provide a bit of context of what's going on here.